core/controllers/payload_validator.py

```diff
@@ -59,6 +64,16 @@ def validate(handler_args, handler_args_schemas, allowed_extra_args):
                 errors.append('Missing key in handler args: %s.' % arg_key)
                 continue
 
+        # Below normalization is for arguments which are expected to be boolean
+        # but from API request they are received as string type.
+        if (
+                allow_string_to_bool_conversion and
+                arg_schema['schema']['type'] == 'bool' and
```

Comment:
Instead of `'bool'` should we use some kind of constant?

Reply:
Added.